sdk/go/examples/anthropic-functions/main.go

```diff
@@ -0,0 +1,112 @@
+/*
+ * This example is part of the Modus project, licensed under the Apache License 2.0.
+ * You may modify and use this example in accordance with the license.
+ * See the LICENSE file that accompanied this code for further details.
+ */
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/hypermodeinc/modus/sdk/go/pkg/http"
+	"github.com/hypermodeinc/modus/sdk/go/pkg/models"
+	"github.com/hypermodeinc/modus/sdk/go/pkg/models/anthropic"
+)
+
+type stockPriceInput struct {
+	Symbol string `json:"symbol"`
+}
+
+// This model name should match the one defined in the modus.json manifest file.
+const modelName = "text-generator"
+
+func GetStockPrice(company string, useTools bool) (string, error) {
+	model, err := models.GetModel[anthropic.MessagesModel](modelName)
+	if err != nil {
+		return "", err
+	}
+	model.Debug = true
+
+	input, err := model.CreateInput(
+		anthropic.NewUserMessage(
+			anthropic.StringContent(fmt.Sprintf("what is the stock price of %s?", company)),
+		),
+	)
+	if err != nil {
+		return "", err
+	}
+	// For Anthropic, system is passed as parameter to the invoke, not as a message
+	input.System = "You are a helpful assistant. Do not answer if you do not have up-to-date information."
+
+	// Optional parameters
+	input.Temperature = 1
+	input.MaxTokens = 100
+
+	if useTools {
+		input.Tools = []*anthropic.Tool{
+			{
+				Name:        "stock_price",
+				InputSchema: `{"type":"object","properties":{"symbol":{"type":"string","description":"The stock symbol"}},"required":["symbol"]}`,
+				Description: "gets the stock price of a symbol",
+			},
+		}
+		input.ToolChoice = anthropic.ToolChoiceTool("stock_price")
+	}
+
+	// Here we invoke the model with the input we created.
+	output, err := model.Invoke(input)
+	if err != nil {
+		return "", err
+	}
+
+	outputs := []string{}
+	for _, content := range output.Content {
```

Comment:
In the AssemblyScript example, we just use `output.content[0]`.  Whichever is best is fine, but they should match.